### PR TITLE
The BEPIS now shows the chance of success in the UI.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
Round 2.5

## About The Pull Request

Some quick changes to the BEPIS UI so that it's more receptive. The UI now guesstimates how close you are to getting a reward at any given amount of banked cash. The value is an estimate based on [Gaussian normal values](http://onlinestatbook.com/2/calculators/normal_dist.html).

image

## Why It's Good For The Game

Dear lord if I get another DM about this.

## Changelog
:cl:
tweak: The BEPIS now shows the odds of success of obtaining a tech in the UI.
/:cl:

### CODE REVIEW

File: tgui-next/packages/tgui/interfaces/Bepis.js [hunk 1]
```diff
@@ -89,6 +89,9 @@ export const Bepis = props => {
             <Box>
 				Average technology cost: {data.mean_value}
             </Box>
+            <Box>
```
Comment (stylemistake) [inline tgui-next/packages/tgui/interfaces/Bepis.js]: Tab alert, I wonder why linter didn't catch that.

File: code/modules/research/bepis.dm [hunk 2]
```diff
@@ -178,19 +178,44 @@
 /obj/machinery/rnd/bepis/ui_data(mob/user)
 	var/list/data = list()
 	var/powered = FALSE
+	var/zvalue = (banked_cash - (major_threshold - positive_cash_offset - negative_cash_offset))/(std)
+	var/std_success = 0
+	var/prob_success = 0
+	//Admittedly this is messy, but not nearly as messy as the alternative, which is jury-rigging an entire Z-table into the code, or making an adaptive z-table.
```
Comment (stylemistake) [inline code/modules/research/bepis.dm]: Calculations were doable in javascript in my opinion, if reasoning was "DM slow and ugly".

**Review by stylemistake — DISMISSED** [review]
Nerdy.

File: code/modules/research/bepis.dm (hunk 2 above)
Comment (ArcaneMusic) [reply code/modules/research/bepis.dm]: If there's a better way to do so, I'm afraid it's out of my skill level I'm afraid. Still, this is certainly better than going in blind when your chance of success is <1%

**Review by stylemistake — APPROVED** [review]

**Pull request merged**